### ISSUE TITLE
URL in GitHub about section for symfony/polyfill-php82 is incorrect

### ISSUE THREAD

**Issue by TimWolla** [symfony/polyfill#407]
![image](https://user-images.githubusercontent.com/209270/168469576-d8e706d7-a72f-4c86-a9e2-abff1f5c2812.png)
It reads `https://symfony.com/polyfill-php72`, but it should probably read `https://symfony.com/polyfill-php82`.

**Comment by derrabus** [comment]
Correct. I don't have the necessary permissions to fix that though. @nicolas-grekas can you look into this?

Also, https://symfony.com/polyfill-php82 yields a 404 at the moment. Maybe @javiereguiluz can help with making that URL work?

**Comment by nicolas-grekas** [comment]
Updated for the GitHub part. Good catch.

**Comment by javiereguiluz** [comment]
The URL is working now: https://symfony.com/polyfill-php82  So, let's close it as fixed.

Thanks @TimWolla for alerting us about these errors.
